pkg/authorization/registry/localresourceaccessreview/rest.go

```diff
@@ -0,0 +1,54 @@
+package localresourceaccessreview
+
+import (
+	"fmt"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kapierrors "k8s.io/kubernetes/pkg/api/errors"
+	"k8s.io/kubernetes/pkg/runtime"
+	kutilerrors "k8s.io/kubernetes/pkg/util/errors"
+	"k8s.io/kubernetes/pkg/util/fielderrors"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	authorizationvalidation "github.com/openshift/origin/pkg/authorization/api/validation"
+	"github.com/openshift/origin/pkg/authorization/registry/resourceaccessreview"
+)
+
+// REST implements the RESTStorage interface in terms of an Registry.
+type REST struct {
+	clusterRARRegistry resourceaccessreview.Registry
+}
+
+// NewREST creates a new REST for policies.
+func NewREST(clusterRARRegistry resourceaccessreview.Registry) *REST {
+	return &REST{clusterRARRegistry}
+}
+
+// New creates a new ResourceAccessReview object
+func (r *REST) New() runtime.Object {
+	return &authorizationapi.LocalResourceAccessReview{}
+}
+
+// Create registers a given new ResourceAccessReview instance to r.registry.
+func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, error) {
```

Comment:
Add to godoc about what this method is doing (transforming a local review into a cluster SAR